exampleSite/content/_index/items-2/extendible.md

```diff
@@ -0,0 +1,7 @@
++++
```

Comment:
extendible -> extensible?

Reply:
Didn't know the difference actually. Thanks.